src/domain/deliveryLog.js

```diff
@@ -0,0 +1,42 @@
+import dbClient from '../utils/dbClient.js'
+
+export default class DeliveryLog {
+  static fromDb(deliveryLog) {
+    return new DeliveryLog(
+      deliveryLog.lines,
+      deliveryLog.date,
+      deliveryLog.userId,
+      deliveryLog.cohortId,
+      deliveryLog.id
+    )
+  }
+
+  static async fromJson(json) {
```

Comment:
Is there a reason this is `async`?

Reply:
used post.js and user.js as a template, that has this static async from json. Should I remove? Or keep it for consistency?

Reply:
As @Alex90Jennings said, other domains are using the async keyword on their static fromJson methods. 
I did try to remove it and I could not store a new DeliveryLog without it.

Reply:
It's not necessary - we can remove the async qualifier from all domain object methods like this one that don't actually perform an async operation.

That work will be in a different PR though.